docs/index.md

```diff
@@ -277,21 +277,43 @@ If you want to add another cluster (like asia-east1 in the above screenshots), r
    metadata:
      name: config-management
    spec:
+     channel: cs
      # clusterName is required and must be unique among all managed clusters
      clusterName: $SHORT_CLUSTERNAME
-     git:
-       syncRepo: git@$GITLAB_HOSTNAME:platform-admins/anthos-config-management.git
-       syncBranch: master
-       secretType: ssh
+     enableMultiRepo: true
+     enableLegacyFields: false
      policyController:
        enabled: true
    EOF
    ```
+
+1. Create a Root Sync configuration for your cluster
+
+   ```shell
+   export NEW_REGION="asia-east1"
+   export SHORT_CLUSTERNAME="prod-${NEW_REGION}"
+   cat > root-sync.yaml <<EOF
+   apiVersion: configsync.gke.io/v1alpha1
+   kind: RootSync
+   metadata:
+     name: root-sync
+     namespace: config-management-system
+   spec:
+     sourceFormat: hierarchy
+     git:
+       repo: git@$GITLAB_HOSTNAME:platform-admins/anthos-config-management.git
+       branch: master
+       auth: ssh
+       secretRef:
+         name: git-creds
+   EOF
+   ```
   
 1. Apply the configuration to your cluster
 
     ```shell
     kubectl apply -f config-management.yaml
```

Comment:
To enable the multi-repo feature. It is used for backward compatibility.

Reply:
This file contains a `ConfigManagement` CR. When it is applied to the cluster with multi-repo enabled, it installs the CRDs and starts a reconciler-manager. Otherwise, the later part doesn't work.

Reply:
I see. Thanks for the explanation.